### PR TITLE
Update .gitattributes

### PR DESCRIPTION
**Pull request by Divyanshu9794**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,15 +1,4 @@
-# See https://github.com/github/linguist about the linguist attributes.
-#
-# See https://help.github.com/en/articles/dealing-with-line-endings
-# about line endings.
-# We expect that you commit text files with Unix line endings to the repository.
```
Comment (thecoder256) [inline .gitattributes:5]: nice bro

**Review by thecoder256 — CHANGES_REQUESTED** [review]
great

**Comment by Divyanshu9794** [comment]
thnx

**Review by thecoder256 — APPROVED** [review]

**Review by thecoder256 — CHANGES_REQUESTED** [review]
changed thats look good

**Review by Divyanshu9794 — COMMENTED** [review]
ok

**Comment by michaliskambi** [comment]
Uhm, I'm not sure what happens here. 

You just created 3 pull requests 
- https://github.com/castle-engine/castle-engine/pull/405
- https://github.com/castle-engine/castle-engine/pull/406
- https://github.com/castle-engine/castle-engine/pull/407

The only changes they do is that they remove (useful and valid) comments from 2 of our files (.gitattributes , fpmake.pp).

You do not provide any explanation for the proposed removal -- PR description is empty, GIT commit logs are meaningless (just saying "Update"). 

The person @thecoder256 made comments in this PR, but they are meaningless. Moreover user https://github.com/thecoder256 was created 6 hours ago, exactly at the time when you created these pull requests -- suggesting this is a sock-puppet (in Wikipedia terminology) you created for the sole purpose of making this PR look valid.

The PR does not do anything useful (on the contrary, it removes useful and valid comments). Rejecting.

**Pull request closed without merging**